SFR/Assembly/outer_assembly/mesh.i

```diff
@@ -0,0 +1,68 @@
+!include ../../common_input.i
+
```

Comment:
include the mesh file for the outer pincells and reuse that information (see LWR examples)